draft-ietf-sedate-datetime-extended.md

```diff
@@ -132,23 +132,22 @@ in {{RFC3339}} that has the following properties:
 
 
 This document does not address extensions to the format where the
-semantic result no longer is a fixed timestamp that is referenced to a
-(past or future) UTC time.
+semantic result is no longer an unambiguous instant in time (past or
```

Comment:
There is a reason that the existing text says "referenced to UTC".
(I have no idea what "unambiguous" means for timestamps, so I would prefer not to use this word.)

Reply:
> There is a reason that the existing text says "referenced to UTC".

Can you articulate that reason? What does it even mean for a timestamp (which is "unambiguous representation of some instant in time", presumably an instance of the `date-time-ext` production) to be "referenced to a UTC time"? The intent of this rewording is to cover hypothetical format extensions where the semantic result of interpreting an instance of `date-time-ext` is "no longer an unambiguous instant in time" but instead describes either no instant in time (such as a local time skipped by DST transition) or multiple instants (such as a local time that is duplicated by DST transition).

Reply:
RFC 3339 is specifically tied to UTC (see its section 1: https://www.rfc-editor.org/rfc/rfc3339#section-1).  This document [hah] is not intended to change this (future ones might add a timescale parameter).